libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.cpp

```diff
@@ -889,7 +889,7 @@ bool HTTPClient::sendHeader(const char * type)
         _base64Authorization.replace("\n", "");
         header += F("Authorization: Basic ");
         header += _base64Authorization;
-        header += "\r\n";
+        header += F("\r\n");
     }
 
     header += _headers + "\r\n";
```

Comment:
At least one more here

Reply:
Since \r\n is used in a bunch of places, it might make sense to define it once as `static` variable at the top of the file (e.g. `crlf`) and reference it from all these places.